### ISSUE TITLE
longitudinal threshold

### ISSUE THREAD

**Issue by MeiHuanshan** [interaction-dataset/INTERPRET_challenge_single-agent#1]
![飞书20210909-090252](https://user-images.githubusercontent.com/29477175/132605237-fbda86b5-0df9-4410-8f0e-c03ac1f3afb7.png)
threshold = -94 when v = 10.9

**Comment by Thomas0Gilles** [comment]
Yes I also believe there is a typo and it should be "1 + (v-1.4)/(11-1.4)", so the value is indeed 2 for v=11 and there is no divergence.

**Comment by bharilya** [comment]
@ZHAN-Wei Dear ZHAN, I am trying to submit my results for the Interaction Single-Agent dataset, but the server returns the following error:

"An error occurred while processing the form. Please try again."

I uploaded a ZIP file containing multiple CSV files for submission. Could you please let me know if there is an issue with the submission format or if I am missing any requirements?
